### PR TITLE
I had an issue on my BO order details page, and this was my fix

### PR DESCRIPTION
**Pull request by michaelhjulskov**
If coinify wasnt used for payment, the dont show anything on the admin order details page.
please let me know if there is a better way to fix

### CODE REVIEW

**Comment by koalalorenzo** [comment]
Related to #3

File: upload/modules/coinify/coinify.php [hunk 1]
```diff
@@ -220,16 +220,18 @@ function readBitcoinpaymentdetails($id_order)
 		{
 			$db = Db::getInstance();
 			$result = $db->ExecuteS('SELECT * FROM `' . _DB_PREFIX_ . 'order_coinify` WHERE `id_order` = ' . intval($id_order) . ';');
-			return $result[0];
+			return (is_array($result[0]))? $result[0] : false;
```
Comment (koalalorenzo) [inline upload/modules/coinify/coinify.php]: Probably a more generic `isset` would work better

**Comment by koalalorenzo** [comment]
Is this also considering previous orders? In that case probably the merchant wants to see more info!

**Comment by michaelhjulskov** [comment]
i have made a PR - hope it is usefull

**Pull request closed without merging**